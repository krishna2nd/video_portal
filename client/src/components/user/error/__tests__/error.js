import React from 'react';
import FieldError from '../';
import renderer from 'react-test-renderer';

test('components/AppContent', () => {
  const component = renderer.create(
    <FieldError error="Some errorr..." ></FieldError>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});