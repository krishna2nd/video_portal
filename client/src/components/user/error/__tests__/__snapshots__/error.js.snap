// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AppContent 1`] = `
<div
  className="error"
>
  <span>
    Some errorr...
  </span>
</div>
`;
